### PR TITLE
Cloudstack - create_node: add param to specify specific ip_address 

### PR DESCRIPTION
**Pull request by boul**
This will set a specific ip_address on the default NIC. 

added tests for this too.

We need this functionality to deploy some instances with the same IP when recreating, this due to specific VPN/Firewall configuration for a particular customer.

### CODE REVIEW

**Comment by Kami** [comment]
LGTM.

/cc @runseb

File: libcloud/compute/drivers/cloudstack.py [hunk 1]
```diff
@@ -836,6 +839,7 @@ def _create_args_to_params(self, node, **kwargs):
         ex_user_data = kwargs.get('ex_userdata', None)
         ex_security_groups = kwargs.get('ex_security_groups', None)
         ex_displayname = kwargs.get('ex_displayname', None)
+        ex_ip_address = kwargs.get('ex_ip_address', None)
```
Comment (sebgoa) [inline libcloud/compute/drivers/cloudstack.py:842]: why do you have ex_ip_address in there but ex_ipaddress in the docstrings ?

**Comment by sebgoa** [comment]
Travis is failing on some minor flake8 issues…do you mind fixing it ?

File: libcloud/compute/drivers/cloudstack.py (hunk 1 above)
Comment (boul) [inline libcloud/compute/drivers/cloudstack.py:842]: something between eyes and keyboard :)

**Comment by boul** [comment]
squashed the commits including the docstring and flake8 fix

**Pull request closed without merging**